cirq/aqt/aqt_device.py

```diff
@@ -26,50 +26,28 @@
 import json
 from typing import Union, Tuple, List, Dict, Sequence, Any, cast
 import numpy as np
-from cirq import ops, devices, study
-from cirq import Circuit, LineQubit, IonDevice, Duration
-from cirq import DensityMatrixSimulator
 
-gate_dict = {'X': ops.X, 'Y': ops.Y, 'Z': ops.Z, 'MS': ops.XX}
+from cirq.ops import (MeasurementGate, X, Y, Z, XX, ZPowGate, XPowGate,
```

Comment:
is there a reason you moved away from `from cirq import ops, ...` pattern.  Google style guide is that you should try to only import packages and modules (https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#s2.2-imports)